targets/target_persistent.c

```diff
@@ -36,17 +80,42 @@ int main(int argc, char* argv[]) {
         base_address, code_start, code_end
     };
 
+    // Configure the stalker with a Transformer and EventSink before prefetching
+    // occurs.
+    //
     GumStalkerTransformer* transformer = gum_stalker_transformer_make_from_callback(
         instr_basic_block, &instr_range, NULL);
+    GumEventSink* event_sink = gum_event_sink_make_from_callback(GUM_COMPILE | GUM_BLOCK,
```

Comment:
```suggestion
    GumEventSink* event_sink = gum_event_sink_make_from_callback(GUM_COMPILE,
```

(GUM_BLOCK will result in a synchronous call to `prefetch_log()` every time any basic block is executed.)